api/v1alpha1/conditions.go

```diff
@@ -67,6 +67,8 @@ const (
 	ReasonFailed string = "Failed"
 
 	ReasonInProgress string = "Progressing"
+
+	ReasonInvalidName string = "InvalidName"
```

Comment:
```suggestion
	ReasonInvalid string = "Invalid"
```

Reply:
Keeping it generic might come in handy in the future since it can be re-used. And we don't really lose any information since you are adding `Message` as well, stating invalid name. Actually could have gotten away with reusing `Failed` as well, your call.